crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -648,6 +672,7 @@ module sui::validator_set {
         get_active_or_pending_or_candidate_validator_mut(self, *validator_cap::verified_operation_cap_address(verified_cap), include_candidate)
     }
 
+    // TODO[ben] - why the next 2 functions are the same?
```

Comment:
They are wrappers that call the same function with slightly different arguments..